templates/zerver/help/enable-high-contrast-mode.md

```diff
@@ -1,25 +1,15 @@
-# Enable high contrast mode for accessibility
+# Enable high contrast mode
 
-Zulip is designed with accessibility in mind. We aim to provide all
-users with the same quality of experience with our product: you can
-read more about our practises in our
-[documentation](https://zulip.readthedocs.io/en/latest/contributing/accessibility.html).
```

Comment:
I think we can leave this pitch in, since it's relevant to a lot of the people that will land on this page.